### PR TITLE
Update Copyright Headers from Facebook to Meta

### PR DESCRIPTION
**Pull request by dillonrg**
- Update file headers across the codebase to include copyright appropriate
 copyright notices. For reference: `Copyright (c) Meta Platforms, Inc. and
 affiliates.`
- Update the `.gitignore` file to ignore idea related directories.

### CODE REVIEW

**Comment by eozturk1** [comment]
LGTM! In some files, we are missing these headers (e.g., *.yml, *.sh). Do we need to add the header to all files?

**Review by eozturk1 — APPROVED** [review]

**Comment by kevinlewi** [comment]
@eozturk1 No, we don't need to add the header to any files that don't already contain the header at the moment.

**Review by slawlor — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -14,3 +14,5 @@ Cargo.lock
 **/target
 
 **/*.log
+
+.idea/**
```
Comment (N0manbutt) [inline .gitignore:18]: 406245a25fca19f50dfd5d153ca1d376feab9466

Comment (N0manbutt) [inline .gitignore:18]:
```suggestion
.idea/**
```

Comment (N0manbutt) [inline .gitignore:18]: +/**

**Review by N0manbutt — COMMENTED** [review]
@zinuzoid

**Comment by N0manbutt** [comment]
> - Update file headers across the codebase to include copyright appropriate
>  copyright notices. For reference: `Copyright (c) Meta Platforms, Inc. and
>  affiliates.`
> - Update the `.gitignore` file to ignore idea related directories.

https://github.com/facebook/akd/commit/ab8d16fd7ebeb8d714bfb3de5b53bd815d9fc65b

**Pull request merged**